kolibri/plugins/device/assets/src/views/ManageTasksPage/index.vue

```diff
@@ -134,11 +121,6 @@
         message: 'Task manager',
         context: 'Title of the page that displays all the tasks in the task manager. ',
       },
-      clearCompletedAction: {
```

Comment:
Are we removing the ability of users to clear completed tasks?

Reply:
Re-added, and added continue to bottom bar for more consistent UI
<img width="904" alt="Screen Shot 2022-04-07 at 12 09 31 PM" src="https://user-images.githubusercontent.com/17235236/162248865-35596b48-ee28-4fe1-87ab-ab5e440ae7d3.png">